plugins/localligator/.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
This should be set to 4.
The whole file should go in the root of the repository.